cardano-testnet/src/Testnet/Process/Cli/Transaction.hs

```diff
@@ -35,6 +39,45 @@ data TxBody
 
 data SignedTx
 
+-- | Calls @cardano-cli@ to sign a simple ADA transfer transaction using
+-- the specified key pairs.
+-- This function takes five parameters:
+-- 
+-- Returns the generated @File TxBody In@ file path to the created unsigned
+-- transaction file.
+buildTransferTx
```

Comment:
`spendOutputsOnlyTx` would be a better name. Transfer is a little ambiguous.

Reply:
Solved [here](https://github.com/IntersectMBO/cardano-node/commit/https://github.com/IntersectMBO/cardano-node/commit/5a98026d8d033684ab97f2cb3291ec365423037d)